recipe/meta.yaml

```diff
@@ -74,11 +87,22 @@ requirements:
     - gnutls
     - gmp
     - glib
-    - freetype        # [linux]
+    - freetype  # [linux]
+    - cairo  # [linux]
+    - gtk3  # [linux]
+    - harfbuzz  # [linux]
```

Comment:
FWIW I double checked and emacs harfbuzz and cairo support are indeed Linux only (they aren't supported on Mac).